app.py

```diff
@@ -452,6 +454,40 @@ def get_first_name(schedule):
     return schedule.get("preferred_name") or schedule["firstname"]
 
 
+def get_latest_github_commits():
+    # get token from secret client
+    secret_client = secretmanager.SecretManagerServiceClient()
+    gh_token = secret_client.access_secret_version(
+        request={"name": "projects/epschedule-v2/secrets/gh_token/versions/1"}
+    ).payload.data
+    # this uses PyGithub module
+    # using an access token from a person who can access epschedule
+    g = gh(gh_token.decode("utf-8"))
+    repo = g.get_repo("EastsidePreparatorySchool/epschedule")
+    # get arr of commits
+    commitsArr = repo.get_commits()
+    # print info about last 3
+    result = []  # initialize array for it
+    for repo_num in range(3):
+        # select the top 3, get its name (title),
+        # author (github name), date, and URL to the changes
+        commit_name = str(commitsArr[repo_num].commit.message).split("\n")[0]
+        commit_author = str(commitsArr[repo_num].commit.author)[16:-2]
+        commit_date = str(commitsArr[repo_num].commit.author.date)
+        commit_url = str(commitsArr[repo_num].html_url)
```

Comment:
`url` should also already be a string

Reply:
fixed all of it on the other branch im about to push in a second